bbb-common-web/src/main/java/org/bigbluebutton/api/MeetingService.java

```diff
@@ -1153,6 +1155,16 @@ public void processGuestPolicyChanged(GuestPolicyChanged msg) {
     }
   }
 
+  public void processPositionInWaitingQueueUpdated(PositionInWaitingQueueUpdated msg) {
+    Meeting m = getMeeting(msg.meetingId);
+    HashMap<String,String> guestUsers = msg.guests;
+    if (m != null) {
+     for (String guestId : guestUsers.keySet()) {
+       m.setWaitingPositionsInWaitingQueue(guestId, guestUsers.get(guestId));
+      }

```

Comment:
I'm trying to understand this, please correct me if I misunderstood.
Won't `PositionInWaitingQueueUpdated` always share the full guest users' list? Couldn't this list just be replaced here instead of copied one by one? What am I missing here?